examples/config-multiple.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "0.1.0-dev",
+    "storage": {
+        "/a": {
+            "rootDirectory": "/tmp/zot1"
```

Comment:
you should probably add at least a "dedupe" flag per entry because let's say we have "dedupe" turned on globally, how does one know if a particular volume/filesystem will not fail on hard links?

Reply:
could you please explain what "dedupe" global option means because now 
``` json 
"storage": {
       "RootDirectory": "/tmp/zot",
       "Dedupe": true
    },